routing/transit_graph.cpp

```diff
@@ -58,13 +58,31 @@ RouteWeight TransitGraph::CalcSegmentWeight(Segment const & segment,
                      0 /* nontransitCross */);
 }
 
+RouteWeight TransitGraph::GetTransferPenalty(Segment const & from, Segment const & to) const
+{
+  if (!IsEdge(from) || !IsEdge(to))
+    return RouteWeight(0 /* weight */, 0 /* nontransitCross */);
+
+  auto lineIdFrom = GetEdge(from).GetLineId();
+  auto lineIdTo = GetEdge(to).GetLineId();
+
+  if (lineIdFrom == lineIdTo)
+    return RouteWeight(0 /* weight */, 0 /* nontransitCross */);
+
+  auto const it = m_transferPenalties.find(lineIdTo);
+  CHECK(it != m_transferPenalties.cend(), ("Segment", to, "belongs to unknown line", lineIdTo));
```

Comment:
`CHECK(it != m_transferPenalties.cend(), ("Segment", to, "belongs to unknown line:", lineIdTo));`?

Reply:
ok